src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java

```diff
@@ -437,25 +484,13 @@ private void processLogicalPortValidation(Map<Integer, LogicalPortInfoEntryV2> a
         }
     }
 
-    private Set<GroupInfoEntryV2> convertGroups(List<GroupSpeakerData> groupEntries, boolean excludeFlowInfo) {
+    private Set<GroupInfoEntryV2> convertGroups(List<GroupSpeakerData> groupEntries) {
         return groupEntries.stream()
-                .map(data -> {
-                    GroupInfoEntryV2 groupEntry = GroupEntryConverter.INSTANCE.toGroupEntry(data);
-                    if (!excludeFlowInfo) {
-                        mirrorGroupRepository.findByGroupIdAndSwitchId(data.getGroupId(), data.getSwitchId())
-                                .ifPresent(mirrorGroup -> {
-                                    String id = mirrorGroup.getFlowId();
-                                    groupEntry.setFlowId(id);
-                                    String pathId = mirrorGroup.getPathId().getId();
-                                    groupEntry.setFlowPathId(pathId);
-                                });
-                    }
-                    return groupEntry;
-                })
+                .map(GroupEntryConverter.INSTANCE::toGroupEntry)
                 .collect(Collectors.toSet());
     }
 
-    private List<MeterInfoEntryV2> convertMeters(SwitchId switchId, List<MeterSpeakerData> meterSpeakerData,
+    private List<MeterInfoEntryV2>  convertMeters(SwitchId switchId, List<MeterSpeakerData> meterSpeakerData,
```

Comment:
hm... is the stylechecker happy with this extra space character?

Reply:
fixed